exec/cnex/cnex.c

```diff
@@ -107,6 +108,7 @@ char *getApplicationName(char *arg)
 int main(int argc, char* argv[])
 {
     gOptions.pszExecutableName = getApplicationName(argv[0]);
+    srand((unsigned int)time(NULL));
```

Comment:
Suggestion: Explicitly call a function like `random_init()` which is implemented in the module code. This one is one line, but others will be more code and you'll want functions.